code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -27,12 +27,11 @@
 		var/list/meancritters = list() // list of possible hostile mobs
 		var/list/nicecritters = list() // and possible friendly mobs
 		for(var/T in typesof(/mob/living/simple_animal))
-			var/mob/living/simple_animal/M = new T
-			switch(M.gold_core_spawnable)
+			switch(initial(T:gold_core_spawnable))

```

Comment:
I'm basing this off of my code here: https://tgstation13.org/phpBB/viewtopic.php?f=5&t=3144 for the verb to find the origin tech of every item.

This code distinctly required :, I remember running into that issue, asking in coderbus and being told i had to use it. 

Oh well, i guess I'll stash my current changes, pull down this pr, and test that